.github/VERSIONING_POLICY.md

```diff
@@ -1,29 +1,34 @@
+These releases are created primarily to contain hot-fix style improvements from prior releases.  Any bugs or security issues identified, or missing UI/UX features from a Minor/Major release might be added to a revision release.  Similar to a Minor release a Revision release will not contain any known breaking changes API.
+
+## API Deprecation Policy (Updated September 2020)
+The DNN Platform project is in a state of transition, continuing to modernize the API and remove existing technology debt.  To this point, it will be necessary for the project to remove/restructuree many public API's.  This will be done methodically, allowing developers to transition away from the older code with time to properly respond to change.
+
+Any API method to be removed will be flagged as deprecated in a release, major, minor or revision, and will be identified to be removed by a specific version.  This will be done using a C# `[Obsolete]` annotation with a comment similar to the following "Deprecated in x.x.x.  Scheduled for removal in vy.0.0, use ____ instead".  The version number of "y" in this example must be 1 major versions ahead of the version in which the notice was added.
 
-## API Deprecation Policy
-The DNN Platform project is in a state of transition, continuing to modernize the API and work towards a transition to .NET Core.  To this point, it will be necessary for the project to remove public API's.  This will be done methodically, allowing developers to transition away from the older code with time to properly respond to change.
-Any API method to be removed will be flagged as deprecated in a release, major, minor or revision, and will be identified to be removed by a specific version.  This will be done using a C# annotation with a comment similar to the following "Deprecated in x.x.x.  Scheduled for removal in vy.0.0, use ____ instead".  The version number of "y" in this example must be 2 major versions ahead.  
-Therefore, an API marked as Deprecated in 9.2.1 can only be removed in version 11.0.  Additionally, methods marked for removal in a version will GUARANTEED be removed in that revision.
+Therefore, an API marked as Deprecated in 9.2.1 can only be removed in version 10.0.  Additionally, methods marked for removal in a version will GUARANTEED be removed in that revision.
 > Example: [Obsolete("Deprecated in DotNetNuke 7.0. This function has been replaced by AddUserRole with additional params. Scheduled removal in v10.0.0.")]
 
+### Testing Recommendations
+It is suggested that all extension developers recompile their projects on the latest API versions on a regular basis to identify removed elements as the compiler warnings will be the primary communication method for these changes.
+
```

Comment:
Should we add here something along the lines of:

For non C# things that may be breaking changes, developers should look at the **Breaking Changes** section of the Dnn release notes of the first major version (example 10.0.0)